ursa.ino

```diff
@@ -20,6 +20,22 @@ rmt_config_t configL;//settings for RMT pulse for stepper motor
 rmt_item32_t itemsL[1];//holds definition of pulse for stepper motor
 rmt_config_t configR;
 rmt_item32_t itemsR[1];
+byte voltage = 0; //0v=0 13v=255
+int signalStrength = -100;
+byte wifI = 0; //counter for wifi sending and recieving
+volatile byte recvdData[50] = {0}; //array to hold data recieved from DS.
+volatile byte recvdDataSize = 0; //how many bytes of the array were actually filled with a message from the DS
+volatile boolean recvdNewData = true;//set true when data gotten, set false when parsed
+volatile byte tosendData[50] = {0}; //array to hold data to send to DS.

```

Comment:
this isn't proper camel case. also, this is just a personal preference, but I think dataToSend sounds better

Reply:
I started off so `tosend` would be parallel to `recvd` but as I was writing the code I kept having to look back at these nemes, so I would support changing these names to something more clear.

Reply:
I fixed this, tell me if it's not ok now